demos/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
```

Comment:
There's example folder, remove demos folder
I would like this index.html in root ghpage (no dev mode)
You add link to dev mode in README.md